DorisProject/src/app/service/categorias.service.ts

```diff
@@ -20,4 +20,17 @@ export class CategoriasService {
     const url = `${environment.apiDoris}${environment.saveCategory}` ;
     return this.http.post(url, category);
   }
+
+  delete(category: any) : Observable <any>{
```

Comment:
Codacy has a fix for the issue: [Replace `·:·Observable·<any>` with `:·Observable<any>·`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9277326)
```suggestion
  delete(category: any): Observable<any> {
```